data/human/deep missions.txt

```diff
@@ -2431,3 +2431,314 @@ event "battle in arneb end"
+			`	And with that, the man disappears into the crowd.`
+				accept
+	on complete
+		dialog `It turns out there is no need to search for Helm, as he is waiting for you when you land.`
+			`	"I figured for you to be returning, it would either be with Skaldgar or with" —he pauses— " news." He looks around. "And I don't see Skaldgar."`
+			`	You show him the odd stone you'd been given, and Helm stiffens then nods. "As I figured. Best we meet in the spaceport to discuss what comes next."`
+
+
+
+mission "Stone of our Fathers 4"
+	name "Stone of our Fathers"
+	description "Take Skaldgar's stone back to his Family-Cairn on <destination>."
+	minor
+	passengers 1
+	to offer
+		has "Stone of our Fathers 3: done"
+	source "Norn"
+	destination "Alfheim"
+	on complete
+		conversation 
+			`The journey to Alfheim was interesting, if uneventful. Thorleif had never been in space before, and every jump left him quite ill. At the same time, however, he found so much wonder in it. Seeing the dark void and bright stars with his own eyes. Watching the tiny worlds in the distance. You could, through his eyes, experience the wild adventure of space anew.`
+			`	Throughout the trip, he told you stories of his family: They were called the Hlewagast and originally settled as some of the first colonists on Alfheim in the early history of The Deep. How they had been explorers and adventurers before taking land here to settle down on and work. How his great grandparents had decided to leave Alfheim, dismayed by the rampant pillaging of resources and looking once more for adventure, and how they had come to Norn. Seeking the waves, and the storms, and the solitude.`
+			`	Through all of this, you got a better feel for Thorleif, Skaldgard and his people, and were almost sorry when your ship approached Alfheim.`
+			`	On landing, you are surprised by how much Alfheim reminds you of Helheim. Though the environment and ecology of the worlds are remarkably different, both are primarily used for resource extraction, and thus filled with boomtown employees and roustabouts, looking for work, chasing dreams, and trying to make their way in the world.`
+			`	Thorleif points to the spaceport. "Let's see if we can find a guide in there. There's a village we'll need to find, assuming it even exists anymore and hasn't been abandoned or destroyed to make room for a new oil rig."`
+	on offer
+		conversation
+			`Helm meets you in the nearly deserted spaceport bar. "Ma' always had a soft spot for Uncle Skaldgar, even after he ran off 35 years ago, she still spoke fondly of him. He always sent the family money, he was good like that. Not as good, though, in sending his love."`
+			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a floating island to the south of here. Are you willing to accompany me?"`
```

Comment:
The description of Norn implies that much of the industry floats, but the islands are regular fixed landmasses. Using habitat felt a bit more appropriate (and gets rid of the mental image I got of an island floating in the air).
```suggestion
			`	Helm sighs, "This is going to be tough, but as bearer of the Stone, you are going to need to present it to his son. His family lives on a floating habitat to the south of here. Are you willing to accompany me?"`
```